doc/source/whatsnew/v1.1.0.rst

```diff
@@ -246,6 +246,18 @@ If needed you can adjust the bins with the argument ``offset`` (a Timedelta) tha
 For a full example, see: :ref:`timeseries.adjust-the-start-of-the-bins`.
 
 
+fsspec now used for filesystem handling
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
this should be in the api breaking section

Reply:
I don't think there are actually breaking changes? For s3fs and gcfs, it stays the same (@martindurant is that correct?), and in enables new filesystems as an enhancement?

Reply:
That's right, for gcs:// and s3://, you get the same file system instances you would have.

Reply:
oh ok (maybe edit to make that clear then)